src/main/java/core/basesyntax/transaction/FruitTransaction.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.transaction;
+
+import core.basesyntax.operation.Operation;
+
+public class FruitTransaction {
```

Comment:
The class `FruitTransaction` is missing documentation. It's a good practice to add a Javadoc comment to explain the purpose of this class and its role within the application.